### DISCUSSION TITLE
Protheus-lib-core v15

### DISCUSSION THREAD

**Discussion by cirqueira-labs** [po-ui/po-angular#1659] [Tire suas dúvidas] [answered]
Olá pessoal.
Existe alguma previsão de liberação do Protheus-lib-core v15 para utilizar com as últimas versões do Angular e PO-UI ?

**Comment by rafaellmarques** [comment, marked as answer]
Boa tarde @cirqueira-labs , tudo bem?

Por favor entre em contato com o time do Protheus para obter informações sobre o protheus-lib-core.
Nosso time não dispõe desses detalhes.

Att.
